bin/ycsb

```diff
@@ -120,12 +122,24 @@ def usage():
 
     return output.getvalue()
 
-def check_output(cmd):
-    p = subprocess.Popen(cmd, stdout=subprocess.PIPE)
-    stdout, _ = p.communicate()
-    if p.returncode:
-        raise subprocess.CalledProcessError(p.returncode, cmd)
-    return stdout
+# From https://gist.github.com/edufelipe/1027906
```

Comment:
we'll need to know the license for the backport. I'll leave a comment on the gist